cmd/celestia-appd/cmd/root.go

```diff
@@ -20,10 +23,11 @@ import (
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	"github.com/cosmos/cosmos-sdk/x/crisis"
 	genutilcli "github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
+	"github.com/go-kit/log"
+	"github.com/rs/zerolog"
 	"github.com/spf13/cobra"
 	"github.com/tendermint/tendermint/cmd/cometbft/commands"
 	tmcli "github.com/tendermint/tendermint/libs/cli"
-	"github.com/tendermint/tendermint/libs/log"
```

Comment:
[question] why does this PR switch from tendermint/libs/log to zerolog and go-kit/log?

Reply:
because this is why the SDK is using when running the node normally.
It was only using this library passing the --log-to-file flag.
This brings consistency between the logger used.